docs/topics/circuit-breaker-mission-intro-paragraph.adoc

```diff
@@ -1,3 +1,6 @@
-Mission proficiency level: xref:proficiency_foundational[*{proficiency-foundational}*].
+Mission proficiency level: 
+//special case since topic is used by front end.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*Foundational*].]
```

Comment:
I'm afraid this still needs to be changed to `{proficiency-foundational}` (in all files).

In addition to that, the URL is hard-coded to point to WF Swarm runtime guide. Is that intentional?